contributing_to_docs/doc_guidelines.adoc

```diff
@@ -257,6 +257,29 @@ link:https://redhat-documentation.github.io/modular-docs/#creating-procedure-mod
 When needed, use `.Prerequisites`, `.Next steps`, or `.Additional resources` syntax to suppress TOC formatting within a module. Do not use `==` headings or xrefs in modules.
 ====
 
+[id="using-conscious-language"]
+== Using conscious language
+
+To assist with the removal of problematic language from the documentation, use the following terminology when referring to OpenShift control plane or compute nodes:
+
+[options="header"]
+|===
+|Branch |Control plane node reference |Compute node reference
+
+|`main` and `enterprise-4.9`
+|Control plane node
+|Compute node
+
+|`enterprise-4.8` and earlier
+|Control plane (also known as master) node
+|Compute (also known as worker) node
```

Comment:
This is the first that I've heard of also using the aka worker for compute nodes too. I thought it was only done for master/control plane?

Reply:
@bergerhoffer thank you for your review! I have now removed the compute node references from this PR.